contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpApiConfig.java

```diff
@@ -131,6 +133,8 @@ public HttpApiConfig(@JsonProperty("url") String url,
 
     this.inputType = inputType == null
       ? "json" : inputType.trim().toLowerCase();
```

Comment:
It is recommended to define "json" as a final static variable.

Reply:
Fixed